src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeedAction.cs

```diff
@@ -104,7 +110,12 @@ public BlobFeedAction(string expectedFeedUrl, string accountKey, MSBuild.TaskLog
             return !Log.HasLoggedErrors;
         }
 
-        public async Task UploadAssets(ITaskItem item, SemaphoreSlim clientThrottle, int uploadTimeout, bool allowOverwrite = false)
+        public async Task UploadAssetsAsync(
```

Comment:
nit: should this be "UploadAssetAsync"?  it looks like it uploads a single item rather than multiple.

Reply:
Yeah, I'll rename it while I'm in here.